elements/rh-dialog/rh-dialog.css

```diff
@@ -1,194 +1,254 @@
 :host {
+  --rh-dialog-backdrop-background-color: rgba(3, 3, 3, 0.62);
+  --rh-dialog-overlay-background-color: transparent;
+
   display: block;
   position: relative;
-
-  --_spacer-align-top: var(--rh-space-md, 8px);
-  --_height-offset: min(var(--_spacer-align-top), var(--rh-space-3xl, 48px));
 }
 
 [hidden] {
   display: none !important;
 }
 
-section {
-  display: flex;
-  position: fixed;
-  height: 100%;
-  width: 100%;
-  top: 0;
-  left: 0;
-  align-items: center;
-  justify-content: center;
-  z-index: 500;
+.visually-hidden {
```

Comment:
> [!NOTE]
> Not relevant to this PR, and please don't commit it to this branch, rather a separate PR.

I think it's time to move this and the previous rule into a shared stylesheet module, which would get adopted (via the static `styles` field) into every element.